RELEASE_NOTES.md

```diff
@@ -1,3 +1,18 @@
+# Version 0.5.9
+
+* Issue #12427 Update ADCP Parsers and Consolidate ADCP stream:
+    * stream "adcp_velocity_earth" replaces:
```

Comment:
Markdown supports formatting such as `adcp_velocity_earth` to denote variable names. Why are you highlighting `adcp_velocity_earth` differently than `adcp_velocity_glider`?

Reply:
becuase adcp_velocity_earth is new stream

Reply:
Quotation marks indicate one of two things:
* direct quotation; she said, "I like spam."
* exact match; you have to spell it "hyperbole."
* facetious; the air quotes, we are going to Long Island Mike's for "salad" and "soda."

I recommend using other formatting to delineate it from other streams, such as bold font.